test-configs.yaml

```diff
@@ -461,6 +461,24 @@ device_types:
     filters:
       - blacklist: {defconfig: ['allnoconfig', 'allmodconfig']}
 
+  meson_g12a_sei510:
```

Comment:
Initial configurations we using underscores instead of dashes, but it's OK in YAML to have dashes.  So to simplify things, you can have the config name set to `meson-g12a-sei510` and drop the `name` property as it's the same.